Python/Product/PythonTools/PythonTools/WorkspaceInfoBarManager.cs

```diff
@@ -128,10 +135,19 @@ class WorkspaceInfoBarManager : IVsRunningDocTableEvents, IDisposable {
                 _condaEnvCreateInfoBar.CheckAsync(),
                 _virtualEnvCreateInfoBar.CheckAsync(),
                 _packageInstallInfoBar.CheckAsync(),
-                _testFrameworkInfoBar.CheckAsync()
+                _testFrameworkInfoBar.CheckAsync(),
+                _pythonVersionNotSupportedInfoBar.CheckAsync()
             );
         }
 
+        private async void TriggerPythonNotSupportedInforBar(object sender, EventArgs e) {
```

Comment:
remove async keyword here

Reply:
You can just call `_pythonVersionNotSupportedInfoBar.CheckAsync().HandleAllExceptions(_serviceProvider, GetType()).DoNotWait();` right here and don't need the extra helper method.